tests/python/pytest_ops.py

```diff
@@ -10,15 +10,42 @@
 
 from benchmarks.python.core import clear_cuda_cache
 from pytest_fusion_definitions import default_fd_fn, parse_inputs_fusion_definition
-from pytest_framework import create_op_test
+from pytest_framework import create_op_test, atexit_serde_create_op_test
 from pytest_core import ReferenceType, OpInfo, SampleInput
 from pytest_opinfos import opinfos
 from pytest_utils import ArgumentType, is_tensor, requiresJAX
 from typing import Callable
 
 from nvfuser import FusionCache, FusionDefinition
 
-from utils import check_captured_python_definition
+from utils import check_captured_python_definition, debug_serde, basic_serde_check
+
+
+def serde_check(test_fn: Callable):
```

Comment:
Can we rename this, maybe `serde_check_ops`.

In PR #2701, I will split the `test_python_frontend.py` file, so it would be easier in avoiding naming conflicts